README.rst

```diff
@@ -29,9 +29,6 @@
 
 #. **Share** experiments and automatically reproduce anyone's experiment.
 
-.. contents:: **Contents**
-  :backlinks: none
```

Comment:
Maybe do it https://github.com/iterative/vscode-dvc style? (I also didn't like that it eats too much space). But overall contents helps a lot also.

Reply:
There is a list of links like that at the top of the DVC readme already, so I tried to combine that with the existing contents. PTAL 🙏 .

Reply:
> But overall contents helps a lot also.

I mostly use GitHub built-in

<img width="459" alt="Captura de Pantalla 2022-07-21 a las 9 01 36" src="https://user-images.githubusercontent.com/12677733/180150751-adea1728-89a4-46a0-8770-745cdc973747.png">